frontend/app/src/composables/defi/lending.ts

```diff
@@ -2,10 +2,8 @@ import { type Balance, BigNumber } from '@rotki/common';
 import { DefiProtocol } from '@rotki/common/lib/blockchain';
 import { assetSymbolToIdentifierMap } from '@rotki/common/lib/data';
 import {
-  AaveBorrowingEventType,
   type AaveHistoryTotal,
-  type AaveLending,
-  isAaveLiquidationEvent
+  type AaveLending
```

Comment:
Can you check if the `isAaveLiquidationEvent` is used anywhere else? if not we should remove it.

Reply:
ok